test/cloud_test.go

```diff
@@ -93,6 +93,32 @@ func TestNodeValueFromMapField(t *testing.T) {
 
 }
 
+func TestPVPriceFromCSV(t *testing.T) {
+	nameWant := "pvc-08e1f205-d7a9-4430-90fc-7b3965a18c4d"
+	pv := &v1.PersistentVolume{}
+	pv.Name = nameWant
+
```

Comment:
You'll want to pull the latest develop, add this line:

```
confMan := config.NewConfigFileManager(&config.ConfigFileManagerOpts{
    LocalConfigPath: "./",
})
```
Then your CSVProvider will need to use the following for `Config`:
```
Config: cloud.NewProviderConfig(confMan, "../configs/default.json"),
```